tests/rules/flowbits/test.yaml

```diff
@@ -0,0 +1,43 @@
+requires:
+    min-version: 7.0.0
+    pcap: false
+
+args:
+    - --engine-analysis
+
+checks:
+- filter:
+    filename: rules.json
+    count: 1
+    match:
+      id: 1
+      lists.packet.matches[0].name: "flowbits"
+      lists.packet.matches[0].flowbits.option: "isset"
```

Comment:
The checks use `flowbits.option`, but the Suri code change has `flowbits.cmd`, which results in a check failure with SV.